src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  background-color: #fff;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+}
+
+.moyo__logo {
+  margin-top: 5px;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  font-family: Roboto, system-ui;
+  font-size: 12px;
+  font-weight: 500;
+  color: black;
+  align-items: center;
+  justify-content: flex-end;
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+  height: 60px;
+  margin-left: 20px;
+  position: relative;
+}
+
+.is-active {
+  margin: 0;
+  position: relative;
+  color: var(--blue-color);
+}
+
+:hover {
+  color: var(--blue-color);
+}
```

Comment:
The :hover pseudo-class should be used with a more specific selector, since this will apply to all elements on the page when hovered. You likely intended to use it only for '.nav__link', so it should be '.nav__link:hover'.